pkg/controller/podautoscaler/horizontal.go

```diff
@@ -1006,21 +1006,21 @@ func calculateScaleUpLimitWithScalingRules(currentReplicas int32, scaleEvents []
 		replicasAddedInCurrentPeriod := getReplicasChangePerPeriod(policy.PeriodSeconds, scaleEvents)
 		periodStartReplicas := currentReplicas - replicasAddedInCurrentPeriod
 		if policy.Type == autoscalingv2.PodsScalingPolicy {
-			proposed = int32(periodStartReplicas + policy.Value)
+			proposed = periodStartReplicas + policy.Value
 		} else if policy.Type == autoscalingv2.PercentScalingPolicy {
 			// the proposal has to be rounded up because the proposed change might not increase the replica count causing the target to never scale up
 			proposed = int32(math.Ceil(float64(periodStartReplicas) * (1 + float64(policy.Value)/100)))
 		}
-		result = selectPolicyFn(result, proposed)
+		steps = append(steps,proposed)
```

Comment:
This variable could be called `proposals`.

Reply:
I'll revise it right away

Reply:
any other suggestions？